lib/web/apps.go

```diff
@@ -74,8 +75,8 @@ func (h *Handler) clusterAppsGet(w http.ResponseWriter, r *http.Request, p httpr
 			Identity:          identity,
 			Apps:              apps,
 		}),
-		StartKey:   resp.NextKey,
-		TotalCount: resp.TotalCount,
+		StartKey:   req.StartKey,
```

Comment:
I think in the Web UI the `StartKey` included in the response is the key that the client is supposed to use as the next `StartKey` if they want to fetch another page. It's a little bit confusing but for now we have to preserve this behavior.